chesster.js

```diff
@@ -644,6 +647,78 @@ controller.hears([
     });
 });
 
+controller.hears([
+    'pairing ([a-zA-Z0-9]+) ([1-8])'
```

Comment:
The first match is the player name right?  If so, you'll need to support more than just characters and letters. @r-mena had a dash in it, and it's not clear to me what other characters are allows. Also, if someone uses: pairing @lakinwecker, slack will translate that into @<USER-ID> which is different from the player name. 

If you use https://github.com/endrawes0/Chesster/blob/master/chesster.js#L97 this function, you can check to see if the word you are matching is a valid user - it will match both by name and id.  You'll have to strip off the @< ... > but otherwise it's pretty straight forward.

I'm starting to wonder if we should have a pre-set regex that we use to match a username or userID and a method that takes both, strips off the necessary characters and returns the appropriate user object.  Probably would be helpful.